### PR TITLE
Add flex-flow and more test case

### PR DESCRIPTION
**Pull request by yisibl**
```
flex-flow: <flex-direction> || <flex-wrap>
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.DS_Store
+*.sublime-workspace
```
Comment (ben-eb) [inline .gitignore]: You're probably better off putting these in your global gitignore. http://stackoverflow.com/a/7335487/1745309

**Comment by ben-eb** [comment]
Awesome, thanks! Could you squash your commits into a single one? :+1:

File: .gitignore (hunk 1 above)
Comment (yisibl) [inline .gitignore]: OK, Thanks, I don't know the trick.

**Comment by yisibl** [comment]
How?

**Comment by ben-eb** [comment]
It's a bit easier if you work on a separate branch, but the general idea is here: http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Comment by ben-eb** [comment]
Done. https://github.com/ben-eb/postcss-ordered-values/commit/6bdb228701e85e595d413f18110807a09195c6b0

**Pull request closed without merging**